apps/music/style/music.css

```diff
@@ -266,12 +290,15 @@ body.picker-mode #title-player {
   overflow-y: auto;
   background: -moz-linear-gradient(center top, #303030 0%, #000000 100%) repeat scroll 0 0 transparent;
   z-index: 10;
-  transition: transform 300ms ease;
+  transition: transform 300ms ease, visibility 300ms;
+  visibility: hidden;
 }
 
 .view {
   width: 100%;
   height: auto;
+  visibility: hidden;
+  transition: visibility 300ms;
```

Comment:
@eeejay What I mean is, does the`visibility` transition actually changes the animations after it's combined with the other existing transitions? cause I try to check with my eye but didn't notice obvious changes on the animations.